src/test/regress/expected/foreign_key_to_reference_table_0.out

```diff
@@ -1,1905 +0,0 @@
- referencing_referencing_table_id_fkey_7000360 | fkey_reference_table.referencing_referencing_table_7000360 | fkey_reference_table.referencing_table_7000352
- referencing_referencing_table_id_fkey_7000361 | fkey_reference_table.referencing_referencing_table_7000361 | fkey_reference_table.referencing_table_7000353
- referencing_referencing_table_id_fkey_7000362 | fkey_reference_table.referencing_referencing_table_7000362 | fkey_reference_table.referencing_table_7000354
- referencing_referencing_table_id_fkey_7000363 | fkey_reference_table.referencing_referencing_table_7000363 | fkey_reference_table.referencing_table_7000355
-(16 rows)
-
-INSERT INTO referenced_table SELECT x, x+1 FROM generate_series(1,1000) AS f(x);
-INSERT INTO referencing_table SELECT x, x+1, x+2 FROM generate_series(1,999) AS f(x);
-INSERT INTO referencing_referencing_table SELECT x, x+1 FROM generate_series(1,999) AS f(x);
-DELETE FROM referenced_table WHERE test_column > 800;
-SELECT max(ref_id) FROM referencing_referencing_table;
- max
----------------------------------------------------------------------
- 800
-(1 row)
-
-DROP TABLE referenced_table CASCADE;
-NOTICE:  drop cascades to constraint fkey_ref on table referencing_table
-DROP TABLE referencing_table CASCADE;
-NOTICE:  drop cascades to constraint referencing_referencing_table_id_fkey on table referencing_referencing_table
-DROP TABLE referencing_referencing_table;
--- test if create_distributed_table works in transactions with some edge cases
--- the following checks if create_distributed_table works on foreign keys when
--- one of them is a self-referencing table of multiple distributed tables
-BEGIN;
-  CREATE TABLE test_table_1(id int PRIMARY KEY);
-  SELECT create_reference_table('test_table_1');
- create_reference_table
----------------------------------------------------------------------
-
-(1 row)
-
-  CREATE TABLE test_table_2(id int PRIMARY KEY, value_1 int, FOREIGN KEY(id) REFERENCES test_table_1(id));
-  SELECT create_distributed_table('test_table_2', 'id');
-NOTICE:  creating a new table for fkey_reference_table.test_table_2
-NOTICE:  Moving the data of fkey_reference_table.test_table_2
-NOTICE:  Dropping the old fkey_reference_table.test_table_2
-NOTICE:  Renaming the new table to fkey_reference_table.test_table_2
- create_distributed_table
----------------------------------------------------------------------
-
-(1 row)
-
-  CREATE TABLE test_table_3(id int PRIMARY KEY, value_1 int, FOREIGN KEY(value_1) REFERENCES test_table_1(id), FOREIGN KEY(id) REFERENCES test_table_2(id));
-ERROR:  cannot create foreign key constraint since foreign keys from reference tables and citus local tables to distributed tables are not supported
-DETAIL:  When adding a foreign key from a local table to a reference table, Citus applies a conversion to all the local tables in the foreign key graph
```

Comment:
We have the same test in `fkeys_between_local_ref`, so no harm to remove alternative output:

```sql
CREATE TABLE local_table_5 (
  col_1 INT UNIQUE REFERENCES distributed_table(col_1),
  FOREIGN KEY (col_1) REFERENCES reference_table_1(col_1));
ERROR:  cannot create foreign key constraint since foreign keys from reference tables and citus local tables to distributed tables are not supported
```